client/scss/components/_icons.scss

```diff
@@ -15,15 +15,12 @@
   color: theme('colors.surface-page');
 }
 
-.icon:after {
-  text-align: end;
-}
-
 // =============================================================================
 // Icon factory methods
 // =============================================================================
 
-$icons-after: ('arrow-down', 'arrow-right', 'arrow-up');
+// Legacy icons still implemented via CSS due to the markup being hard to change.
+$icons-after: ('arrow-down', 'arrow-up');
```

Comment:
We still need `arrow-right` accessible via CSS elsewhere in the code (jQuery Datetimepicker), but we don’t need it as a "universally reusable" `icon-arrow-right-after` like this suggests.